tree_sitter_v/grammar.cjs

```diff
@@ -18,7 +18,6 @@ const PREC = {
   or: 1,
   resolve: 1,
   composite_literal: -1,
-  empty_array: -2,
   strictly_expression_list: -3,
```

Comment:
should not removing `  empty_array: -2,`, make `strictly_expression_list` be -2 now?

Reply:
ok